site/src/components/introduction/building-a-chart-complete.js

```diff
@@ -0,0 +1,87 @@
+// a random number generator
+var generator = fc.data.random.walk()
+    .steps(11);
+
+var valueformatter = d3.format('$f');
+var dateFormatter = d3.time.format('%b');
+
+// create the data
+var data = {
+    // target values for the annotations
+    targets: [{
+        name: 'low',
+        value: 8.5
+    }, {
+        name: 'high',
+        value: 11.2
+    }],
+    // randomly generated sales data
+    sales: generator(5).map(function(d, i) {
```

Comment:
I know accurate data isn't the point here, but it looks like about half the year's total sales happen in January if the chart shows a cumulative sum.
I'm wondering if this is would be a good opportunity to have an example that uses d3.json / csv? This has been requested previously (#764), and seems very common in other d3 examples.